.clang-format

```diff
@@ -3,7 +3,11 @@
 BasedOnStyle: WebKit
 AccessModifierOffset: -3
 AllowShortCaseLabelsOnASingleLine: true
-BreakBeforeBraces: Allman
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterControlStatement: false
```

Comment:
Not sure why you want to change this. The clang-format is not that old and the codebase is being normalized slowly as the code is refactored/updated (we decided against a global format update). Whanging the clang-format now would mean that a large part of the codebase would have to be updated and that it will take longer to get consistent formatting in the project.

Reply:
Because @jsm174 told me the rules were different.

Reply:
There are no official rules. I was just asking you to keep the code similar to what we had in every other PUP file. 
I don't have my editor try to automatically format my code.. That stuff never works 100%.

Reply:
No problem, just please, don't modify the clang-format file as it will makes things a bit more difficult in the src folder where I have autoformatting enabled in MSVC

Reply:
reverted